RetrospectiveExtension.Frontend/components/feedbackBoardContainer.tsx

```diff
@@ -1194,7 +1200,7 @@ class FeedbackBoardContainer extends React.Component<FeedbackBoardContainerProps
                             items: this.boardActionContexualMenuItems,
                           }}
                         >
-                          <span className="ms-Button-icon"><i className="fas fa-ellipsis-h"></i></span>&nbsp;
+                          <span className="ms-Button-icon"><i className="fa-solid fa-grip"></i></span>&nbsp;
```

Comment:
**question**: Is there a reason that this was changed to "grip"?

I thought that the `...` pattern was well-established, and the `:::` icon was for drag-and-drop affordances.

Just curious :)

Reply:
If this was inadvertent, I've created #456 to fix it